zenodo/config.py

```diff
@@ -357,6 +357,54 @@ def _(x):
 GITHUB_PID_FETCHER = 'zenodo_doi_fetcher'
 #: GitHub metdata file
 GITHUB_METADATA_FILE = '.zenodo.json'
+#: GitHub error handlers
+GITHUB_ERROR_HANDLERS = [
+    (
+        'zenodo.modules.deposit.errors.VersioningFilesError',
+        'zenodo.modules.gitbub.error_handlers.versioning_files_error'
```

Comment:
```suggestion
        'zenodo.modules.github.error_handlers.versioning_files_error'
```

Same for entries below